OpenSDS_Volume_Actions_Design.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Status: Pending
+
+Version: Beta
+
+Author(s): PengYiOfOpenSDS
+
+## Summary
+
+Extends the size of a volume. 
+
+## Motivation
+
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Goals
+
+What will be addressed by the proposal?
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Non-Goals
+
+What will not be addressed by the proposal?
+"Shrinks the size of a volume" will not be addressed.
+
+## Proposal
+
+Extends the size of a volume to a requested size, in gibibytes (GiB). 
+
+### Data model impact
+
+Does the proposal affect data model?
```

Comment:
This placeholder line can be removed, then just change the next line from "No" to "None".